src/org/zaproxy/zap/extension/viewstate/HttpPanelViewStateView.java

```diff
@@ -0,0 +1,269 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2010 psiinon@gmail.com
```

Comment:
Copied from old source, if you have an example of what it (and other examples of the same) should be updated to I will do so